src/scheduler/workers/processListsBeforeAndDuringStart/getEmailTypeForToday.ts

```diff
@@ -0,0 +1,43 @@
+import type { CurrentAnnualReview } from "shared/types";
+import type { AnnualReviewKeyDates } from "server/models/types";
+import { startOfToday } from "date-fns";
+import type { RemindersBeforeStartDate } from "scheduler/batch/helpers";
+
+/**
+ * Determines which email should be sent based on today's date.
+ */
+export function getEmailTypeForToday(keyDates: CurrentAnnualReview["keyDates"]): RemindersBeforeStartDate | undefined {
+  if (!keyDates?.annualReview) {
+    return undefined;
```

Comment:
Out of curiosity if you just wrote `return` without the undefined would that return undefined?

Reply:
it would, yep, dunno why I've done so! I don't usually go for explicit undefined or null value tbh